socialhome/streams/app/components/Stream.vue

```diff
@@ -1,13 +1,13 @@
 <template>
     <div>
         <div class="container-flex">
-            <div id="new-content-container" class="hidden">
-                <a href="" id="new-content-load-link" onclick="return false;">
+            <div v-show="$store.state.stream.hasNewContent" id="new-content-container">
+                <b-button @click.prenvent.stop="onNewContentClick" class="new-content-load-link">
```

Comment:
How does this look visually? Can have screenshot? Current looks like this:

![selection_404](https://user-images.githubusercontent.com/1174866/29496523-dfb751a8-85dc-11e7-95c2-2b54ac4f4bb9.png)

A button seems like it could be very blunt looking.